<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsDialog</class>
 <widget class="QDialog" name="OptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	background-color: rgb(26, 26, 26);
}</string>
  </property>
  <widget class="QLineEdit" name="defaultPath">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	font: 600 11pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
	padding-left: 7px;
	letter-spacing: 1px;
}</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="doneBtn">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>430</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 600 12pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
}

QPushButton::hover {
	background-color: rgb(84, 84, 84);
}</string>
   </property>
   <property name="text">
    <string>Done</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fontBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Verdana Pro Cond Semibold</family>
     <pointsize>14</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
	font: 600 14pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 4px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
}</string>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>false</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="specialValueText">
    <string/>
   </property>
   <property name="minimum">
    <number>3</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
  </widget>
  <widget class="QLabel" name="fontSizeLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 600 16pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
	letter-spacing: 1px;
}</string>
   </property>
   <property name="text">
    <string>Font Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="resetBtn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>430</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	font: 600 12pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
}

QPushButton::hover {
	background-color: rgb(84, 84, 84);
}</string>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </widget>
  <widget class="QLabel" name="textColorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 600 16pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
	letter-spacing: 1px;
}</string>
   </property>
   <property name="text">
    <string>Text Color</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="backColorLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 600 16pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
	letter-spacing: 1px;
}</string>
   </property>
   <property name="text">
    <string>Back Color</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="fontStyleLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
	font: 600 16pt &quot;Verdana Pro Cond Semibold&quot;;
	color: rgb(218, 218, 218);
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
	letter-spacing: 1px;
}</string>
   </property>
   <property name="text">
    <string>Font Style</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFontComboBox" name="fontStyleBox">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>185</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QFontComboBox {
	font: 600 9pt &quot;Verdana Pro Cond Semibold&quot;;
	border: 5px solid rgb(39, 39, 39);
	background-color: rgb(59, 59, 59);
	color: rgb(218, 218, 218);
	padding-left: 5px;
}</string>
   </property>
  </widget>
  <widget class="QPushButton" name="currentTextColorLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>260</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="currentBackColorLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>81</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
